### DISCUSSION TITLE
How to add alt tag to <x-smart-image>?

### DISCUSSION THREAD

**Discussion by pandeydip** [dietercoopman/smart#20] [Q&A] [answered]
As for SEO its good to use `alt` tag in image, how can I use `alt` tag with `<x-smar-image ... > `component?

**Comment by dietercoopman** [comment, marked as answer]
👀 Have you tried the `alt` tag 👀  

It works just like this 

```html
<x-smart-image src="smart.png" width="200px" alt="this is a title" />
```

👉 Tip: To create a tooltip for an image, use the title attribute!

kind regards Dieter

**Reply by pandeydip** [reply]
@dietercoopman Thanks for your answer that did really work. Found that I was using quotes in the wrong place. Thanks.
